pandas/tests/io/formats/test_format.py

```diff
@@ -1699,7 +1699,7 @@ def test_to_string_format_na(self):
         tm.reset_display_options()
         df = DataFrame(
             {
-                "A": [np.nan, -1, -2.1234, 3, 4],
+                "A": [np.nan, -1, -2.1_234, 3, 4],
```

Comment:
Same here, so this would be ``-2.123_4``. Actually I'm not sure underscores after the dot here aid understanding...

Reply:
After looking the web for the past 25 minutes.
I came to a conclusion that you're 100% correct.

Even python it self doesn't put underscores after the decimal point.

```python
>>> '{:_d}'.format(123456789)
'123_456_789'
>>> '{:_f}'.format(123456789.123456789)
'123_456_789.123457'
```